src/VirtoCommerce.News.Web/App/src/modules/vc-news/pages/news-article-details.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+  <VcBlade v-loading="loading" :title="$t('VC_NEWS.PAGES.DETAILS.TITLE')" :expanded="expanded" :closable="closable"
+    width="70%" :toolbar-items="bladeToolbar" @close="$emit('close:blade')" @expand="$emit('expand:blade')"
+    @collapse="$emit('collapse:blade')">
+    <VcContainer>
+      <VcForm class="tw-flex tw-flex-col tw-gap-4">
+        <Field name="name" :model-value="newsArticle.name" :label="$t('VC_NEWS.PAGES.DETAILS.FORM.NAME.LABEL')"
```

Comment:
To make validation work properly, you need to use the v-slot in Field like this:

```
<Field v-slot="{ errorMessage, handleChange, errors }">
  <VcInput
    :error="!!errors.length"
    :errorMessage="errorMessage"
    @update:model-value="handleChange"
  />
</Field>
```
The way you implemented it won’t display validation messages because:

errorMessage and errors must come from the Field slot scope;

:error should be bound to !!errors.length (not errors.length directly);

handleChange ensures the value updates correctly and triggers validation.